pkg/webhooks/validation/controller/controller.go

```diff
@@ -170,37 +171,13 @@ func makeHandler(queue workqueue.Interface, gvk schema.GroupVersionKind) *cache.
 			if skip {
 				return
 			}
-			req := &reconcileRequest{
+			req := reconcileRequest{
 				event:       updateEvent,
+				webhookName: currObj.GetName(),
 				description: fmt.Sprintf("update event (%v, Kind=%v) %v", gvk.GroupVersion(), gvk.Kind, key),
 			}
 			queue.Add(req)
 		},
-		DeleteFunc: func(curr interface{}) {
```

Comment:
Deleted this since I don't believe we need to reconcile on deletion and it caused race conditions when switching revision tag on default validator